doc/streaming-roadmap.md

```diff
@@ -1,42 +1,38 @@
-# Overview
+# Streaming Roadmap
 
 This document describes the road-map for gRPC-Web to support different streaming features.
 * Server-streaming
 * Client-streaming and half-duplex streaming
-* Full-duplex streaming over HTTP
-* Full-duplex streaming over Web-transport
+* Full-duplex streaming over WebTransport
 
-# Server-streaming
+## Server-streaming
 
 We will keep improving server-streaming in the following areas:
-* Finalizing keep-alive support (via Envoy) - 2020
-* Performance improvements and whatwg Fetch/streams support, including service workers - 2020
-* Addressing runtime behavior gaps between Fetch and XHR - 2020+
+* Fetch cancellation support - 2023
+* Finalizing keep-alive support (via Envoy) - 2023+
+* Performance improvements and whatwg Fetch/streams support, including service workers - 2023+
+* Addressing runtime behavior gaps between Fetch and XHR - 2023+
 
-# Client-streaming and half-duplex streaming
-
-We are actively involved in the Chrome [Origin Trial](https://developers.chrome.com/origintrials/#/view_trial/3524066708417413121) 
-to finalize the fetch/upload stream API spec (whatwg). One of the pending issues that blocks the final spec is to decide whether it is safe to enable
-upload-streaming over HTTP/1.1. We believe that upload-streaming should be enabled for both HTTP/2 and HTTP/1.1. Specifically for gRPC-Web, the server can't control 
-the client deployment. As a result, if upload-streaming is only enabled over HTTP/2, a gRPC service will have to implement a non-streaming method 
-as a fallback for each client-streaming method.
+## Client-streaming and half-duplex streaming
 
 Once the whatwg spec is finalized and upload-streaming is supported by major browsers (2020), we will add client-streaming support
```

Comment:
How should i rewrite this sentence? This seems outdated since it's still referring to 2020.